### ISSUE TITLE
Charging elapsed time display is incorrect due to time zone issues

### ISSUE THREAD

**Issue by sandeen** [OpenEVSE/ESP8266_WiFi_v2.x#170]
When RAPI tells me elapsed time is 1144s, the HTML display tells me "18:19:04" instead of the correct 00:19:04.  This is because of time zone issues apparently, and the way that elapsed time is handled:

`self.elapsedNow(new Date((openevse.status.elapsed() * 1000) + ((new Date()) - self.elapsedLocal())));`

I can't quite tell what this is trying to do, to be honest... but the 18 hours above seems to be because I am at GMT-6:00, so 00 hours GMT is 18H in my time zone.

Would using `new Date(0,0,0,0,0,1144,0)` give the right answer for an elapsed time, which should not care about time zones?  I admit I get confused w/ javascript time, browser time, time zones, and the like.

**Comment by glynhudson** [comment]
Thanks for reporting. I tired setting my PC's time to -6hrs but I'm unable to replicate. Please could you post the output of  

http://openevse.local/status

**Comment by sandeen** [comment]
`{"mode":"STA","wifi_client_connected":1,"srssi":-83,"ipaddress":"10.0.0.103","emoncms_connected":1,"packets_sent":39772,"packets_success":37285,"mqtt_connected":1,"ohm_hour":"NotConnected","free_heap":23144,"comm_sent":594703,"comm_success":594666,"amp":0,"pilot":24,"temp1":157,"temp2":0,"temp3":230,"state":1,"elapsed":13853,"wattsec":31999214,"watthour":1006230,"gfcicount":0,"nogndcount":0,"stuckcount":0,"divertmode":1}`
<img width="475" alt="image" src="https://user-images.githubusercontent.com/427124/38120027-83831ba8-338b-11e8-9bb7-e8d4ea10e2c8.png">

**Comment by sandeen** [comment]
(hm, weird that divertmode:1 but the slider is off?)

**Comment by glynhudson** [comment]
Thanks, I can see the issue. 

Divertmode 1 is means normal charge. While divert mode 2 is 'eco' mode:

https://github.com/OpenEVSE/ESP8266_WiFi_v2.x/blob/bd140684f42d7e8a678f450b8edd3b82b5db87cc/src/divert.cpp#L29

**Comment by sandeen** [comment]
Oops sorry.  Base-9 brain here ;)

**Comment by sandeen** [comment]
Sorry - after an update to 2.7.0 and a restart, now I don't see it.   Somehow I had missed this change:

```
-    return addZero(dt.getHours())+":"+addZero(dt.getMinutes())+":"+addZero(dt.getSeconds());
+    return addZero(dt.getUTCHours())+":"+addZero(dt.getMinutes())+":"+addZero(dt.getSeconds());
```
which was kinda buried in "bf14d0fbRTC detect complete"

**Comment by glynhudson** [comment]
@sandeen this issue should be now fixed in a better way in the latest version V2.7.1. Please could you test and confirm. Thank you 

https://github.com/OpenEVSE/ESP8266_WiFi_v2.x/releases/tag/2.7.1
